third_party/beam/sdks/go/pkg/beam/runners/disksort/gbk_test.go

```diff
@@ -22,13 +22,18 @@ import (
 	"github.com/apache/beam/sdks/go/pkg/beam"
 	"github.com/apache/beam/sdks/go/pkg/beam/testing/passert"
 	"github.com/apache/beam/sdks/go/pkg/beam/testing/ptest"
+	"github.com/apache/beam/sdks/go/pkg/beam/x/debug"
 )
 
+// Disable parallelism to work with order-dependent passert checks.
+func init() { *gbkConcurrency = 1 }
+
 func TestGBK(t *testing.T) {
 	p, s, nums := ptest.CreateList([]int{1, 2, 3, 4, 2, 3, 3})
 
 	kv := beam.ParDo(s, extendToKey, nums)
 	sums := beam.DropKey(s, beam.ParDo(s, sum, beam.GroupByKey(s, kv)))
+	debug.Print(s, sums)
```

Comment:
Is this leftover or useful?

Reply:
They're useful since the `passert` package just panics on a failure.